### PR TITLE
Update the default issuers/purses

### PR DESCRIPTION
**Pull request by michaelfig**
Some refactoring and additions.

### CODE REVIEW

File: packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js [hunk 1]
```diff
@@ -304,18 +317,30 @@ export function buildRootObject(vatPowers, vatParameters) {
           additionalPowers.treasuryCreator = treasuryCreator;
         }
 
+        const mintNames = [];
+        const mintPurses = [];
```
Comment (dtribble) [inline packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js]: rename to have "name" in this variable?  otherwise the usage is hard to parse.

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js [hunk 2]
```diff
@@ -21,76 +20,227 @@ export const CENTRAL_ISSUER_NAME = 'MOE';
...
- * @property {string} pursePetname
- * @property {Bigish} mintValue
- * @property {Array<[Bigish, Bigish]>} [fakeTradesGivenCentral]
+ * @property {Array<[string, Bigish]>} [defaultPurses]
+ * @property {Array<[Bigish, Bigish]>} [tradesGivenCentral]
  */
 
+export const makeScaler = toDecimals => (n, fromDecimals = 0) => {
+  if (typeof n === 'bigint') {
+    return n * 10n ** BigInt(toDecimals);
```
Comment (dtribble) [inline packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: This should use NAT rather than BigInt?

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js [hunk 3]
```diff
@@ -21,76 +20,227 @@ export const CENTRAL_ISSUER_NAME = 'MOE';
...
  * @property {CollateralConfig} [collateralConfig]
- * @property {string} pursePetname
- * @property {Bigish} mintValue
- * @property {Array<[Bigish, Bigish]>} [fakeTradesGivenCentral]
+ * @property {Array<[string, Bigish]>} [defaultPurses]
+ * @property {Array<[Bigish, Bigish]>} [tradesGivenCentral]
  */
 
+export const makeScaler = toDecimals => (n, fromDecimals = 0) => {
+  if (typeof n === 'bigint') {
```
Comment (dtribble) [inline packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: shouldn't this be checking that `fromDecimals` is 0?

Comment (dtribble) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: perhaps he method needs a comment

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js [hunk 4]
```diff
@@ -21,76 +20,227 @@ export const CENTRAL_ISSUER_NAME = 'MOE';
...
+    {
+      issuerArgs: [undefined, { decimalPlaces: 6 }],
+      defaultPurses: [['Cosmos Staking', scaleMicro(68)]],
+      collateralConfig: {
+        keyword: 'ATOM',
+        collateralValue: scaleMicro(1000000n),
+        initialMarginPercent: 150n,
+        liquidationMarginPercent: 125n,
+        interestRateBasis: 250n,
+        loanFeeBasis: 50n,
```
Comment (dtribble) [inline packages/cosmic-swingset/lib/ag-solo/vats/issuers.js:107]: Dont these need an initial price?

Comment (dtribble) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js:107]: IS that `collateralValue`?

File: packages/treasury/bundles/install-on-chain.js [hunk 5]
```diff
@@ -100,8 +100,8 @@ export async function installOnChain({ agoricNames, board, chainTimerService, na
     [instanceAdmin, treasuryUiDefaults.AMM_NAME, ammInstance],
     [brandAdmin, 'TreasuryGovernance', govBrand],
     [issuerAdmin, 'TreasuryGovernance', govIssuer],
-    [brandAdmin, 'MOE', moeBrand],
-    [issuerAdmin, 'MOE', moeIssuer],
+    [brandAdmin, 'RUN', centralBrand],
```
Comment (dtribble) [inline packages/treasury/bundles/install-on-chain.js]: "RUN" should come from config or constant?

**Review by dtribble — APPROVED** [review]
Some improvements to be made but nothing blocking.  Mostly getting "RUN" from already-present constants.

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js (hunk 3 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: Done.

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js (hunk 4 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js:107]: Initial price comes from `tradesGivenCentral[0]`.  I've added a comment.

File: packages/treasury/bundles/install-on-chain.js (hunk 5 above)
Comment (michaelfig) [reply packages/treasury/bundles/install-on-chain.js]: Done, now a `centralName` property in the function call's options.

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js (hunk 2 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: Done.

File: packages/cosmic-swingset/lib/ag-solo/vats/issuers.js (hunk 3 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/vats/issuers.js]: Actually, the separate branch is for bigint values specifically (for which `fromDecimals` doesn't matter).  The other logic handles the case where a `number` is specified, regardless of `fromDecimals` value.

I've added comments to that effect.
